### PR TITLE
[MIG/MOVE]14.0 product_variant_inactive

### PR DESCRIPTION
**Pull request by PierrickBrun**
This module was in product-attribute in v12.0 but I think it belongs here.

### CODE REVIEW

File: product_variant_inactive/tests/test_button.py [hunk 1]
```diff
@@ -64,5 +64,5 @@ def test_create_variant_do_not_reactivate(self):
         change the "active" state of the existing variant"""
         product = self.env.ref("product.product_product_4")
         product.active = False
-        product.product_tmpl_id.create_variant_ids()
-        self.assertEquals(product.active, False)
+        product.product_tmpl_id._create_variant_ids()
+        self.assertEqual(product.active, False)
```
Comment (rousseldenis) [inline product_variant_inactive/tests/test_button.py]: You can use assertFalse() but nitpicking

**Review by rousseldenis — APPROVED** [review]
LGTM

File: product_variant_inactive/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def _compute_product_variant_count_all(self):
+        for rec in self:
```
Comment (rousseldenis) [inline product_variant_inactive/models/product_template.py:10]: And maybe a test for the count_all field

**Review by andreampiovesana — APPROVED** [review]

**Pull request merged**